File: Treinamento/C/Exercícios_listas/Questão03.c
#include <stdio.h>

int main() {
    double notas[4];
    double soma = 0;

    for (int c = 0; c < 4; c++) {
        printf("Digite a %i° nota: ", c + 1);
        scanf("%lf", &notas[c]);
    }

    for (int c = 0; c < 4; c++) {
        soma += notas[c];
    }

    double media = soma / 4.0;
    printf("Média: %lf\n", media);
}

File: Treinamento/Java/Exercícios_listas/Questão03.java
import java.util.Scanner;
public class Questão03 {
    public static void main(String[] args) {
        double vet_notas[] = new double[4];
        double soma = 0;
        Scanner s;
        s = new Scanner(System.in);

        for (int c = 0; c < 4; c++) {
            System.out.print("Digite a " + (c + 1) + "° nota: ");
            double nota = s.nextDouble();
            vet_notas[c] = nota;
        }

        for (int c = 0; c < 4; c++) {
            soma += vet_notas[c];
        }
   
        System.out.printf("Média: %.2f\n", (soma / 4.0));
    }
}
